src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
        private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
If you create a formatter, make it a constant (https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names). Let's move this formatter to fields.